_posts/2018-08-06-Kickstarter-Trajectory-2018-Summer.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+authors: [wout_meskens, sander_smets, michael_dewree, steven_deleye, ken_keymolen]
+title: "Kickstarter Trajectory 2018 Summer Edition"
+image: /img/kicks.png
+tags: [Spring, Spring Boot, Angular, Unit Test, Mocking, Microservices, Git, DevOps, Docker, Typescript, Kickstarter, Security]
+category: Kickstarters
+comments: true
+---
+
+## Introduction
+>54 young professionals started the Ordina Kickstarter trajectory, on the 1st of August, this summer. 
+JWorks counted 5 kickstarters: Sander, Steven, Ken, Wout en Michaël. All looking for a challenge and a fresh start. 
+Some just graduated from school, others already had multiple years of experience, IT related or not. 
+The goal was to broaden every kickstarter’s knowledge of the fast evolving IT world. 
+We learned the basics of every hot topic which will definitely give us enough credits for our first project.
+
+The kickstarter trajectory consisted of 2 parts: 
+* 1 month covered with all kinds of trainings: technical, non-technical, backend, frontend, devops...  
+* After our minds were filled with all this information, there was a devcase where we could put everything we learned into practice.
+
+## First Day
+On the first day of the kickstarter trajectory we were welcomed into the company and got an introduction about the structure of the company. 
+After that we took a tour around the building where we were told what the different workspaces are and where the different teams work. 
+It was nice to notice that everyone we met was very friendly and helpful, this made us feel directly at ease.
+On the first day we also received the keys of our car and a laptop so we were equipped to begin our journey at Ordina.
+
+## Security
+In the beginning of the trajectory we got an introduction of the most important security principles also including information security like GDPR. 
+This is very important to Ordina because all its consultants need to always have this information in the back of their minds when working at a customer.
+We also followed a more technical security course which explained some of the most common attacks and how to avoid them.
+
+## Backend
+
+### Java
+In this lecture given by [Yannick De Turck](/author/yannick-de-turck/){:target="_blank"}, we were introduced to all new features of Java 7 and 8 aswell as Java 9.
```

Comment:
We also saw Java 10. ;-)
https://github.com/ordina-jworks/kickstarter-trajects/blob/master/2018/kickstarter-traject-summer/java-7-8-9-10/java-10-jworks-yannick-de-turck.pdf